shelley/design-spec/delegation_design_spec.tex

```diff
@@ -2596,16 +2611,16 @@ \subsection{Parameters}
 \item
   The desired number of pools \(k\in\mathbb{N}_+\).
 \item
-  The influence \(a_0\in[0,\infty)\) a pool leader's stake should have
-  on the desirability of the pool. Small values of \(a_0\) indicate
-  little influence.
+  The influence \(a_0\in[0,\infty)\) the stake pledged by the owners should have
+    on the desirability of the pool. Small values of \(a_0\) indicate little
+    influence.
 \item
   The \emph{expansion rate} \(\rho\in[0,1]\), determining the fraction
-  of still available ADA that will be created per epoch.
+  of still available ada that will be created per epoch.
 \item
   The fraction \(\tau\in[0,1]\) of rewards going to the treasury.
-\item
-  An exponent \(\gamma\in(0,\infty)\) for penalty calculation.
+%% \item
```

Comment:
Any reason why we're leaving this comment here?

Reply:
Nope, no reason. I'll delete it :)